src/components/views/login/InteractiveAuthEntryComponents.js

```diff
@@ -142,7 +143,7 @@ export const PasswordAuthEntry = React.createClass({
                         { submitButtonOrSpinner }
                     </div>
                 </form>
-                <div className="error">
+                <div className="error" role={ this.props.errorText ? "alert" : ""}>
```

Comment:
Again, I think it'd be easier to read if the existence of the error div was conditional on the `errorText` and the `role` was always "alert".

Reply:
Changed all of InteractiveAuthEntryComponents to display the error div conditionally.